experiments/integrated/global/global_parameters.jl

```diff
@@ -0,0 +1,210 @@
+using ArtifactWrappers
+#currently not being used
```

Comment:
It looks like this is being used by `global_soil_canopy.jl`! Could you add a comment that this file is meant to be used with that simulation (or others like it)?

Reply:
oh, maybe you meant the runoff section, not the whole file :) 

do you plan to include runoff in this PR?

Reply:
I opened PR #669 to do so, and yes, I meant just the runoff section